### PR TITLE
try fix safari in mobile

### PR DESCRIPTION
**Pull request by Idan-Berman**

### CODE REVIEW

**Review by yanivfrancoTs — APPROVED** [review]

File: webauthn-for-logged-out-users/pages/login.html [hunk 1]
```diff
@@ -43,7 +43,12 @@
       }
 
       async function passkeyModal() {
-        await abortPasskeyAutofill();
+        const userAgent = window.navigator.userAgent;
+        console.log('userAgent: ', userAgent);
+        if (!userAgent.includes('safari')) {
```
Comment (ts-ron) [inline webauthn-for-logged-out-users/pages/login.html:48]: what about chrome?

Comment (Idan-Berman) [reply webauthn-for-logged-out-users/pages/login.html:48]: chrome for sure needs to do abort. still checking. we have some issue with safari on mobile but still not sure why. 
that is the easiest way for me to check this

Comment (ts-ron) [reply webauthn-for-logged-out-users/pages/login.html:48]: Hmmm maybe I do not understand the code - but in the hosted solution (which we now also use for work login) chrome works well with auto-fill. What am I missing?

Comment (Idan-Berman) [reply webauthn-for-logged-out-users/pages/login.html:48]: autofill works fine. modal doesn't. we think someohow the abort affecting it in mobile safari

**Pull request merged**